src/DonationForms/Blocks/DonationFormBlock/resources/editor/components/DonationFormBlockPreview.tsx

```diff
@@ -44,7 +42,13 @@ export default function DonationFormBlockPreview({
             {openFormButton}
         </a>
     ) : isModalDisplay ? (
-        <ModalForm dataSrc={`/?givewp-route=donation-form-view&form-id=${formId}`} embedId={''} openFormButton={openFormButton} isFormRedirect={false} formViewUrl={''} />
+        <ModalForm
+            dataSrc={`/?givewp-route=donation-form-view&form-id=${formId}`}
+            embedId={''}
+            buttonText={openFormButton}
+            isFormRedirect={false}
+            formViewUrl={''}
+        />
```

Comment:
Let's add an `unreleased` tag in the top of this export to document this change.

Reply:
Resolved: c8a8942326eaefd89f8fb3c7432292a20c6ffd6d